### PR TITLE
Electron 77 - added alwaysontop option for windows installer

### PR DESCRIPTION
**Pull request by VishwasShashidhar**
@lneir Please review and merge.

CC: @VikasShashidhar

### CODE REVIEW

**Comment by VishwasShashidhar** [comment]
Merged with the latest master

File: installer/win/Symphony-x64.aip [hunk 1]
```diff
@@ -161,7 +162,7 @@
     <ROW Path="&lt;AI_DICTS&gt;ui_en.ail"/>
   </COMPONENT>
   <COMPONENT cid="caphyon.advinst.msicomp.DigCertStoreComponent">
-    <ROW DigitalCertificate="Wrapper.Windows.Product.RSA.pkcs12.pfx" TimeStampUrl="http://timestamp.verisign.com/scripts/timstamp.dll" SignerDescription="[|ProductName]" DescriptionUrl="https://www.symphony.com" SignOptions="7" SignTool="0" UseSha256="1" Password="4A99BAA4D493EE030480AF53BA42EA11CCFB627AB1800400DA9692073D68C522A10A4FD0B5F78525294E51AC7194D55B5EE1D31F"/>
+    <ROW DigitalCertificate="Wrapper.Windows.Product.RSA.pkcs12.pfx" TimeStampUrl="http://timestamp.verisign.com/scripts/timstamp.dll" SignerDescription="[|ProductName]" DescriptionUrl="https://www.symphony.com" SignOptions="0" SignTool="0" UseSha256="1" Password="4A99BAA4D493EE030480AF53BA42EA11CCFB627AB1800400DA9692073D68C522A10A4FD0B5F78525294E51AC7194D55B5EE1D31F"/>
```
Comment (lneir) [inline installer/win/Symphony-x64.aip]: looks like something changed with signing?

**Comment by lneir** [comment]
only need to commit changes in .aip file.  other files aren't needed.

**Comment by VishwasShashidhar** [comment]
@lneir Fixed, please review.

**Pull request merged**